InvenTree/InvenTree/settings.py

```diff
@@ -958,6 +958,9 @@
 # Are plugins enabled?
 PLUGINS_ENABLED = get_boolean_setting('INVENTREE_PLUGINS_ENABLED', 'plugins_enabled', False)
 
+# Are sample plugins enabled?
+PLUGINS_SAMPLES = get_boolean_setting('INVENTREE_PLUGINS_SAMPLES', 'plugins_samples', False)
```

Comment:
Question: This will greatly reduce the exposure of users to sample plugins - which is the easiest way to discover the possibilities - when in debug mode. Is this intended?

Reply:
This was mostly because I was sick of the sample plugins clogging up the plugin screen. I run a lot of instances in debug!

What I will do instead is get the API filter working properly, and then sample plugins can be hidden from the table.